packages/react-dom/src/events/DOMEventProperties.js

```diff
@@ -79,6 +79,10 @@ const discreteEventPairsForSimpleEventPlugin = [
 
 const otherDiscreteEvents = [
   DOMTopLevelEventTypes.TOP_CHANGE,
+  // Ensure we still treat "focus" and "blur" as
+  // discrete events.
```

Comment:
Why can't we delete them altogether? What is their use now?

Reply:
Good point. I guess early on in this PR, we didn't know if we could remove focus and blur entirely. We believe we can now, so it's all good. :)